test/sqlsrv/srv_037_sqlsrv_has_rows.phpt

```diff
@@ -0,0 +1,103 @@
+--TEST--
+sqlsrv_has_rows() using a forward and scrollable cursor
+--SKIPIF--
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+
+// Connect
+$conn = sqlsrv_connect( $serverName, $connectionInfo);
+if( !$conn ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Create database
+sqlsrv_query($conn,"CREATE DATABASE ". $dbName) ?: die();
+
+// Create table
+$query = "CREATE TABLE $tableName (ID VARCHAR(10))";
```

Comment:
after running the script twice, it doesn't reproduce the issue anymore because the table exist and the number of values gets doubled. better to use $dbName to create the table, or drop table at the end of the test